mllib-tests/v1p5/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -322,7 +333,7 @@ abstract class RecommendationTests(sc: SparkContext) extends PerfTest {
     val implicitRatings: Boolean = booleanOptionValue(IMPLICIT)
 
     val data = DataGenerator.generateRatings(sc, numUsers, numProducts,
-      math.ceil(numRatings * 1.25).toLong, implicitRatings,numPartitions,seed)
+      math.ceil(numRatings).toLong, implicitRatings,numPartitions,seed)
```

Comment:
The ALS DataGenerator is different, though.  It already generates 1.25x the data since it creates train & test jointly to avoid issues with new IDs in the test data.  So this is currently creating ```1.25^2``` the data.